packages/documentation/docs/controls/_toast_styleguide.md

```diff
@@ -0,0 +1,48 @@
+Toasts are small pop-ups that provide simple feedback on a process. They are UI elements where an event causes a small text field to appear screen. Toasts are informative, only last a few seconds, and only take up a very small part of the screen to avoid interrupting the workflow. They usually follow an action performed by the user and provide information about the success or failure of that action. We typically use toasts for immediate feedback or tips on actions that a user performs, e.g., successful deletion.
```

Comment:
```suggestion
Toasts are small pop-ups that provide simple feedback on a process. They are UI elements where an event causes a small text field to appear on screen. Toasts are informative, last for a few seconds only, and take up a very small part of the screen to avoid interrupting the workflow. They usually follow an action performed by the user and provide information about the success or failure of that action. We typically use toasts for immediate feedback or tips on actions that a user performs (e.g. successful deletion).
```